Umbraco-Cloud/Troubleshooting/Deployments/Type-Not-Found/index.md

```diff
@@ -0,0 +1,45 @@
+---
+versionFrom: 7.0.0
+---
+
+# Extraction error: "Type not found! "
+
+This issue will present itself as an extraction error on your target environment for a deploy.
+
+![Extraction error on Live](images/Error_Environment.png)
+
+The extraction error happens with two Document Types where one of them have been allowed as a child node type for  the other Document Type.
+
+If the Document Type that have been allowed as the child is then deleted,
+
+the references for the child Document Type is not removed from the parent Document Type.
+
+Because the parent Document Type still have the references from the deleted child Document Type,
+
+the environment will throw an extraction error telling us that it can not find the Document Type that is allowed as a child
+
+when trying to deploy to the next environment.
```

Comment:
```suggestion
Because the parent Document Type still has the references from the deleted child Document Type, the environment will throw an extraction error when trying to deploy to the next environment. The error will tell us that it can not find the Document Type that is allowed as a child.
```